packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -388,7 +389,9 @@ export default Mixin.create(EmberArray, MutableEnumerable, {
     @public
   */
   addObject(obj) {
-    if (!this.contains(obj)) {
+    var included = isEnabled('ember-runtime-enumerable-includes') ? this.includes(obj) : this.contains(obj);
```

Comment:
Unfortunately the ternary syntax is not supported by feature flag stripping. Please rewrite this to use an `if` statement.